network_generation.py

```diff
@@ -39,4 +41,20 @@ def six_bus(vn_high=20, vn_low=0.4, length_km=0.03, std_type='NAYY 4x50 SE'):
                    name='line5')
     pp.create_line(net, 5, 7, length_km=length_km, std_type=std_type,
                    name='line6')
+    #  add controllable storage
+    num_batteries = np.shape(battery_locations)[0]
+    for i in range(num_batteries):
+        pp.create_storage(net, bus=battery_locations[i], p_kw=0.0, max_e_kwh=energy_capacity, soc_percent=init_soc)
```

Comment:
Let's align this with `NetModel.add_battery`. There are additional fields that need to be included, as well as non-standard fields.